tests/e2e/config.py

```diff
@@ -0,0 +1,65 @@
+import json
+import re
+import subprocess
+from typing import Literal, Protocol, cast
+
+import boto3
+
+
+class EndToEndTestSecrets(Protocol):
+    @property
+    def GOVUK_NOTIFY_API_KEY(self) -> str: ...
+
+
+class LocalEndToEndSecrets:
+    @property
+    def GOVUK_NOTIFY_API_KEY(self) -> str:
+        with open(".env") as env_file:
```

Comment:
I wonder if we want to use pydantic-settings to try and keep this consistent everywhere, I'm not too particular about it

Reply:
interesting thought - may revisit 🤔